js/media.view.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file media.view.js
+ */
+(function ($, Drupal) {
+
+  "use strict";
+
+  /**
+   * Registers behaviours related to view widget.
+   */
+
+  Drupal.behaviors.MediaLibraryView = {
+    attach: function (context, settings) {
+      var $view = $('.view-content grid', context);
+      $view.once('media').prepend('<div class="grid-sizer"></div><div class="gutter-sizer"></div>');
+
+      // Indicate that images are loading.
+      $view.append('<div class="ajax-progress ajax-progress-fullscreen">&nbsp;</div>');
+      $view.imagesLoaded(function () {
+        $view.masonry({
+          columnWidth: '.grid-sizer',
+          gutter: '.gutter-sizer',
+          itemSelector: '.grid-item',
+          percentPosition: true,
+          isFitWidth:true
+        });
+        // Add a class to reveal the loaded images, which avoids FOUC.
+        $('.grid-item').addClass('item-style');
+        $view.find('.ajax-progress').remove();
+      });
+
+      $('.grid-item').once('bind-click-event').click(function () {
+        var input = $(this).find('.views-field-entity-browser-select input');
```

Comment:
This is not a file browser reference. views-field-entity-browser-select is the class of the select element of entities on the modal.